src/integrators/integrator_utils.jl

```diff
@@ -233,7 +233,7 @@ end
 function stepsize_controller!(integrator,alg::Union{StandardControllerAlgs,
                               OrdinaryDiffEqNewtonAdaptiveAlgorithm{:Standard}})
   # Standard stepsize controller
-  if EEst == zero(EEst)
+  if integrator.EEst == zero(integrator.EEst)
```

Comment:
Out of curiosity: why not just `iszero(integrator.EEst)` since then types could specialize on this and the fallback is `integrator.EEst == zero(integrator.EEst)` anyway (see https://github.com/JuliaLang/julia/blob/36e21d47b5bea100c54e4a4d1be845cbfa83a669/base/number.jl#L40)? I've seen these comparisons multiple times in JuliaDiffEq and I'm really interested in whether there's a reason to use them instead of `iszero` :smiley:

Reply:
I didn't know about `iszero` 😃